example/example-node.ts

```diff
@@ -10,7 +10,7 @@ async function example() {
     const tezos = new TezosToolkit(provider);
 
     console.log('Getting storage...');
-    await tezos.contract.at('KT1W1jh5C5NbcVVvpnBLQT9ekMbR5a8fg6mc').then(async contract => { // knownBigMapContract used in integration tests
+    await tezos.contract.at('KT1VyQwS4oZAdYxuWmMNqJ6CtJFtXQbLZDjJ').then(async contract => { // knownBigMapContract used in integration tests
```

Comment:
The contract address should be changed at line 24 also, but this is not the right contract for the example (line 26 will fail).
This contract can be used: https://github.com/ecadlabs/taquito/blob/master/integration-tests/config.ts#L61

Reply:
updated the contract address